packages/wdio-crossbrowsertesting-service/tests/service.test.ts

```diff
@@ -156,17 +156,21 @@ describe('wdio-crossbrowsertesting-service', () => {
         const cbtService = new CrossBrowserTestingService({}, [], {})
         cbtService.browser = browser
         cbtService.failures = 0
-
-        cbtService.afterTest({}, {}, { passed: true })
+        const testResult = {
+            passed: true
+        } as WebdriverIO.TestResult
+        cbtService.afterTest({} as WebdriverIO.Test, {} as any, testResult)
         expect(cbtService.failures).toEqual(0)
     })
 
     it('afterTest: failed test', () => {
         const cbtService = new CrossBrowserTestingService({}, [], {})
         cbtService.browser = browser
         cbtService.failures = 0
-
-        cbtService.afterTest({}, {}, { passed: false })
+        const testResult = {
+            passed: false
+        } as WebdriverIO.TestResult
+        cbtService.afterTest({} as WebdriverIO.Test, {} as any, testResult)
```

Comment:
I've seen we do this multiple times, let's just create a test object and pass it along in `afterTest` in all the tests